clustered/integration-test/build.gradle

```diff
@@ -60,6 +60,8 @@ task copyServerLibs(type: Copy) {
 
 test {
   dependsOn copyServerLibs
+
+  jvmArgs "-Xmx8g", "-XX:+UseG1GC", "-XX:MaxDirectMemorySize=64g"
```

Comment:
For me this passes fine without the tunings. Are we sure we need this?

Reply:
yeah.  i get OOME without it on OversizedCacheOpsPassiveTest

Reply:
That test has been a thorn in my side for a while. Could you `@Ignore` it, remove the tuning and I'll look at it separately?

Reply:
ok

Reply:
Ignored.